packages/core/src/compatibility/browser.ts

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+ * Licensed under the BSD-3 License as modified (the “License”); you may obtain a copy
+ * of the license at https://github.com/palantir/blueprint/blob/master/LICENSE
+ * and https://github.com/palantir/blueprint/blob/master/PATENTS
+ */
+
+const userAgent = navigator.userAgent;
+const browser = {
+    isEdge: !!userAgent.match(/Edge/),
+    isInternetExplorer: (!!userAgent.match(/Trident/) || !!userAgent.match(/rv:11/)),
+};
+
+export const Browser = {
+    isEdge: () => browser.isEdge,
+    isInternetExplorer: () => browser.isInternetExplorer,
```

Comment:
why are these functions that return constants? instead of constants themselves? 

(remember you can't modify constants, by definition)